<?xml version="1.0" ?>
<launch>
  <node name="save_sensorimg" pkg="save_sensorimg" type="save_sensorimg_node" output="screen" >
 
   <param name="video_device" value="/dev/video6" />
    <param name="image_width" value="1600" />
    <param name="image_height" value="1200" />
    <param name="pixel_format" value="mjpeg" />
    <param name="framerate" value="30" />
    <param name="camera_frame_id" value="usb_cam" />

    <param name="io_method" value="mmap"/>
    <param name="brightness" value="-1"/>
    <param name="contrast" value="-1"/>
    <param name="saturation" value="-1"/>
<!--
   <param name="brightness" value="32"/>
    <param name="contrast" value="32"/>
    <param name="saturation" value="64"/>
-->
    <param name="sharpness" value="-1"/>

    <param name="autofocus" value="false"/>
    <param name="focus" value="-1"/>
    <param name="autoexposure" value="false"/>
    <param name="exposure" value="20"/>
    <param name="auto_white_balance" value="false"/>
    <param name="white_balance" value="4600"/>

    <param name="io_method" value="mmap"/>



  </node>
<!--
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image" to="/save_sensorimg/image_raw"/>
    <param name="autosize" value="true" />
  </node>
-->
  <node name="save_sensordata" pkg="save_sensorimg" type="save_sensorimg_save" output="screen">
  </node>
</launch>
